src/WMStream.ts

```diff
@@ -0,0 +1,29 @@
+const util = require('util');
+const stream = require('stream');
+const Writable = stream.Writable;
+
+export default class WMStream extends Writable 
+{
+  _write: any;
```

Comment:
Is the underscore required? Seem to be inconsistent that on line 9 `key` isn't using such convention. consider removing underscore and just use `private` as that have been the convention so far in other repos.